### PR TITLE
Readme changes for v1.2

### PR DESCRIPTION
**Pull request by mshariq-nerd**
## What do these changes do?
1. Readme changes for 1.2.0 release

## Are there changes in behavior for the user?
No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,8 +62,8 @@ See details [here](windows-nginx-deployment-guide.md)
 Browser | Tested Version | Supported
 --------|-------- |-------
 Safari (mac)  | 11.0.x   | latest 1
-Chrome  |65.0.x (64-bit) | latest 2 
-Firefox |59.0.x (64-bit) | latest 2
+Chrome  |666.0.x (64-bit) | latest 2
```
Comment (monika-Dianomic) [inline README.md]: 666? should be `66.x`

File: README.md [hunk 2]
```diff
@@ -62,8 +62,8 @@ See details [here](windows-nginx-deployment-guide.md)
 Browser | Tested Version | Supported
 --------|-------- |-------
 Safari (mac)  | 11.0.x   | latest 1
-Chrome  |65.0.x (64-bit) | latest 2 
-Firefox |59.0.x (64-bit) | latest 2
+Chrome  |666.0.x (64-bit) | latest 2 
+Firefox |60.0 (64-bit) | latest 2
```
Comment (monika-Dianomic) [inline README.md]: `60.x`

File: README.md [hunk 3]
```diff
@@ -62,8 +62,8 @@ See details [here](windows-nginx-deployment-guide.md)
 Browser | Tested Version | Supported
 --------|-------- |-------
 Safari (mac)  | 11.0.x   | latest 1
```
Comment (monika-Dianomic) [inline README.md]: `11.x`

File: README.md [hunk 4]
```diff
@@ -72,4 +72,4 @@ Firefox |59.0.x (64-bit) | latest 2
 [Developer's Guide](developers-guide.md)
 
 
-> &copy; 2017 DIANOMIC SYSTEMS. All Rights Reserved.
\ No newline at end of file
+> &copy; 2017 DIANOMIC SYSTEMS. All Rights Reserved.
```
Comment (monika-Dianomic) [inline README.md]: 2017-18 for year

File: changelog.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-## v1.2 (under development)
+## v1.2
```
Comment (monika-Dianomic) [inline changelog.md]: v1.2.0

File: developers-guide.md [hunk 6]
```diff
@@ -30,20 +30,20 @@ Set API base URL in `environments/environment.prod.ts`, you can always change it
...
+ TypeScript   | 2.3.4         |        2.8.3          | Apache 2.0
+ rxjs         | 5.4.3         |        6.1.0          | Apache 2.0
 
 #### Dev:
  Library         |   Version     | Latest Stable (? Y/n) | License
 ---------------- | ------------- | --------------------  | ------------
-chart.js         |  2.7.0        |        2.7.0          | MIT 
-core-js          |  2.5.1        |        2.5.1          | MIT 
-lodash-es        |  4.17.7       |        4.17.4         | MIT
+chart.js         |  2.7.0        |        2.7.2          | MIT
```
Comment (monika-Dianomic) [inline developers-guide.md]: should we upgrade to 2.7.2 itself? as its bug fixes release

Comment (mshariq-nerd) [reply developers-guide.md]: Yes we can do that. I have checked it and it is working fine.

**Pull request merged**